src/Makefile

```diff
@@ -65,7 +68,9 @@ IOS_CORE_SOURCES += $(IOS_EXTRA_SOURCES)
 IOS_SOURCES += $(IOS_EXTRA_SOURCES)
 
 IOS_GENERATOR_FLAGS = -inline-selectors -d:IOS -process-enums -warnaserror:$(IOS_GENERATOR_WARNASERROR)
+IOS_GENERATOR_native_FLAGS = -d:XAMCORE_2_0 -d:__UNIFIED__
```

Comment:
minor: `native` in caps - but if I understand correctly where you're going this is temporary :)

Reply:
This native I believe has to match the native we pass in here:

https://github.com/xamarin/xamarin-macios/blob/master/src/Makefile#L158